src/main/drivers/serial_uart_stm32f10x.c

```diff
@@ -124,7 +124,10 @@ uartPort_t *serialUART1(uint32_t baudRate, portMode_t mode, portOptions_t option
     // UART1_RX    PA10
     if (options & SERIAL_BIDIR) {
         IOInit(IOGetByTag(IO_TAG(PA9)), OWNER_SERIAL, RESOURCE_UART_TXRX, 1);
-        IOConfigGPIO(IOGetByTag(IO_TAG(PA9)), IOCFG_AF_OD);
+        if (options & SERIAL_BIDIR_PP)
```

Comment:
Use conditional assignment, eg
```
IOConfigGPIO(IOGetByTag(IO_TAG(PA9)), options & SERIAL_BIDIR_PP ? IOCFG_AF_PP : IOCFG_AF_OD);
```
It's more readable and makes it obvious that only the second parameter changes.